src/workflows/multiomics/rna_multisample/config.vsh.yaml

```diff
@@ -87,13 +87,20 @@ functionality:
           multiple_sep: ','
           required: false
           default: [50, 100, 200, 500]
+  dependencies:
+    - name: transform/normalize_total
+    - name: transform/log1p
+    - name: filter/filter_with_hvg
+    - name: qc/calculate_qc_metrics
```

Comment:
I have updated the workflows to include the following (see 5833fee):

QC statistics are now only calculated once where needed. This means that the mitochondrial gene detection is performed in the `rna_singlesample` pipeline and the other count based statistics are calculated during the `prot_multisample` and `rna_multisample` pipelines. In both cases, the `qc` pipeline is being used, but only parts of that workflow are activated by parametrization. Previously the count based statistics were calculated in both the `singlesample` and `multisample` pipelines, with the results from the multisample pipelines overwriting the previous results.

The reason why the `rna_multisample` pipeline did not use the qc pipeline is because the qc pipeline is in fact mitochondrial gene detection + basic count qc calculation (calculate_qc_metrics). In the multisample workflow, only the basic count qc statistics are needed, so it was enough to use `calculate_qc_metrics`. 

However, I think it is cleaner to just use the `qc` pipeline everywhere so I changed to make sure that it is possible to disable the the basic count qc calculations and mitochondrial gene detection (either one or both), providing more flexibility.